src/sgame/sg_bot.cpp

```diff
@@ -514,6 +514,22 @@ void G_BotInit()
 	{
 		InitTreeList( &treeList );
 	}
+
+	// default bot names
```

Comment:
I don't think we should hard code names here.

Reply:
Where is the best place to do that? Or is there another way (like a default config file shipped with the game)?

Reply:
I don't think we have one. We don't have a script that runs by default. We could modify the code here

https://github.com/Unvanquished/Unvanquished/blob/master/src/sgame/sg_bot.cpp#L44

to have a default list of names, and then switch between the hardcoded list and the custom one when someone calls G_AddBotNames() or something. I'm happy with just Bot#1, etc.

Reply:
Doing `Bot#1` means duplicating a lot of code for prefix handling and padding computation. So it's a good idea but I don't want to include this within this PR. It can be done later.